### ISSUE TITLE
MailPreview Missing Route in Mail Template

### ISSUE THREAD

**Issue by swiffer** [cakephp/debug_kit#600]
Tried using the MailPreview feature and added a Preview method for welcoming the user as shown in the instructions.

However I've added a link to the Login page within my `welcome.ctp` Mail Template file

```php
$this->Html->link(__('Complete Account creation'), ['_full' => true, 'controller' => 'Users', 'action' => 'login']);
```

What i get now is:

Error: A route matching "array ( 'controller' => 'Users', 'action' => 'login', 'plugin' => 'DebugKit', '_ext' => NULL, )" could not be found.

**Comment by swiffer** [comment]
changing that line of code to

```php
$this->Html->link(__('Complete Account creation'), ['_full' => true, 'controller' => 'Users', 'action' => 'login', 'plugin' => null]);
```
fixes the problem, is this intended?

**Comment by josegonzalez** [comment]
Yeah you need to specify the plugin, unfortunately. CakePHP's routing will otherwise default keys to the current scope. You'd have a similar issue if you somehow used a UsersController template in a different plugin but didn't specify the controller.

**Comment by markstory** [comment]
We might be able to remove the `plugin` attribute of the current request context while mail previews are rendered.
